src/main/java/formflow/library/FormFlowController.java

```diff
@@ -64,12 +88,27 @@ protected FlowConfiguration getFlowConfigurationByName(String flow) {
     return flowConfigurationList.get(0);
   }
 
+  /**
+   * Checks if there are any flows matching the String parameter.
+   *
+   * @param flow {@link String} of a flow name.
+   * @return {@link Boolean} True if the flow is found in the FlowConfiguration.  False if the flow is {}not found in the flowConfiguration.
+   */
   protected Boolean doesFlowExist(String flow) {
```

Comment:
The @return has a `is {}not found` which seems odd. Also, I think you can capitalize the `f` in `flowConfiguration` on this same line.

Reply:
done

Reply:
The `<br>` makes this look really nice!